pep-0707.rst

```diff
@@ -0,0 +1,380 @@
+<https://github.com/python/cpython/pull/30122>`_ and
+`for the raised exception in 3.12
+<https://github.com/python/cpython/pull/101607>`_. As a consequence,
+several APIs that expose the triplet were deprecated in favour of
+simpler alternatives:
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * -
+     - Deprecated
+     - Alternative
+   * - Get handled exception (Python)
+     - :func:`py3.12:sys.exc_info`
+     - :func:`py3.12:sys.exception`
+   * - Get handled exception (C)
+     - :c:func:`py3.12:PyErr_GetExcInfo`
+     - :c:func:`py3.12:PyErr_GetHandledException`
+   * - Set handled exception (C)
+     - :c:func:`py3.12:PyErr_SetExcInfo`
+     - :c:func:`py3.12:PyErr_SetHandledException`
+   * - Get raised exception (C)
+     - :c:func:`py3.12:PyErr_Fetch`
+     - :c:func:`py3.12:PyErr_GetRaisedException`
+   * - Set raised exception (C)
+     - :c:func:`py3.12:PyErr_Restore`
+     - :c:func:`py3.12:PyErr_SetRaisedException`
+   * - Construct an exception instance from the 3-tuple (C)
+     - :c:func:`py3.12:PyErr_NormalizeException`
+     - N/A
+
+
+The current proposal is a step in this process, and considers the way
+forward for one more case in which the 3-tuple representation has
+leaked to the language. The motivation for all this work is twofold.
+
+Simplify the implementation of the language
+-------------------------------------------
+
+The simplification gained by reducing the interpreter's internal
+representation of the handled exception to a single object was significant.
+Previously, the interpreter needed to push onto/pop
+from the stack three items whenever it did anything with exceptions.
+This increased stack depth (adding pressure on caches and registers) and
+complicated some of the bytecodes. Reducing this to one item
+`removed about 100 lines of code <https://github.com/python/cpython/pull/30122>`_
+from ``ceval.c`` (the interpreter's eval loop implementation), and it was later
+followed by the removal of the ``POP_EXCEPT_AND_RERAISE`` opcode which has
+become simple enough to be `replaced by generic stack manipulation instructions
+<https://github.com/python/cpython/issues/90360>`_.  Micro-benchmarks showed
+`a speedup of about 10% for catching and raising an exception, as well as
+for creating generators
+<https://github.com/faster-cpython/ideas/issues/106#issuecomment-990172363>`_.
+To summarize, removing this redundancy in Python's internals simplified the
+interpreter and made it faster.
+
+The performance of invoking ``__exit__``/``__aexit__`` when leaving
+a context manager can be also improved by replacing a multi-arg function
+call with a single-arg one. Micro-benchmarks showed that entering and exiting
+a context manager with single-arg ``__exit__`` is about 13% faster.
+
+Simplify the lanaguage itself
+-----------------------------
+
+One of the reasons for the popularity of Python is its simplicity. The
+:func:`py3.11:sys.exc_info`` triplet is cryptic for new learners,
```

Comment:
```suggestion
:func:`py3.11:sys.exc_info` triplet is cryptic for new learners,
```

Fix one other typo from me that I hadn't yet added to the previous review before it was randomly submitted